app/plugins/Overview/js/overview.js

```diff
@@ -3,25 +3,28 @@
 const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
+// Ensure precision
+BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ EXPONENTIAL_AT: 1e+9 });
```

Comment:
Can you explain the reasoning for the exponential at 1e9? Will it ever even be displayed this way?

I guess it'll make more sense once I see it in action